connector/src/main/java/org/geysermc/connector/network/translators/java/JavaStatisticsTranslator.java

```diff
@@ -33,14 +33,13 @@
 
 @Translator(packet = ServerStatisticsPacket.class)
 public class JavaStatisticsTranslator extends PacketTranslator<ServerStatisticsPacket> {
-
```

Comment:
Pretty sure in the translators we've consistently kept this whitespace. I've gotten hooked on no whitespace for fields but I don't think it looks clean for methods.